### DISCUSSION TITLE
vercel source code issue

### DISCUSSION THREAD

**Discussion by PritamSarkar12212004** [vercel/community#6545] [Bug Report] [answered]
### Page to Investigate

https://wish-box-full-stack.vercel.app/

### Steps to Reproduce

iam facing vercel source code output issue 

!
![Capture](https://github.com/vercel/community/assets/162453679/40c1ed10-ee7f-419d-99d5-1ce7c3424571)

{
  "version": 2,
  "builds": [
    {
      "src": "./api/index.js",
      "use": "@now/node"
    }
  ]
}

**Comment by versecafe** [comment]
@PritamSarkar12212004  try something like this
```json
{
  "functions": {
    "api/index.js": {
      "memory": 1024,
      "maxDuration": 10
  }
}
```

**Reply by PritamSarkar12212004** [reply]
Not working

**Reply by PritamSarkar12212004** [reply]
Plz clone my repo and make valid code for deploy vercel

**Reply by versecafe** [reply]
can you share your repo

**Reply by PritamSarkar12212004** [reply, marked as answer]
Its working

[a comment by an automated account is not shown]
